src/main/resources/vocabularies/wikidata/wikidata.xsl

```diff
@@ -256,4 +571,63 @@
 
     </xsl:template>
 
+    <xsl:function name="lib:isAcceptableLang" as="xs:boolean">
+        <xsl:param name="string"/>
+
+        <xsl:value-of select="contains($langs,lower-case($string))"/>
+    </xsl:function>
```

Comment:
@hugomanguinhas for your information: our new library for running XSLT files accepts this file, but gives the following warning:
```
Warning at function lib:isAcceptableLang on line 574 
  SXWN9000: A function that computes atomic values should use xsl:sequence rather than xsl:value-of
```
Is this something you need to look at or change? Or is it good to go?